.changelog/unreleased/features/provider/2035-min-stake-max-rank.md

```diff
@@ -1,2 +1,3 @@
-- Add minimum stake requirement to let consumer chains
-  determine requirements for validators that validate their chain. ([\#2035](https://github.com/cosmos/interchain-security/pull/2035))
+- Add `min_stake`, a power shaping configuration that enables consumers to set 
```

Comment:
Mention that `min_stake` this is a parameter?
Similarly below for `allow_inactive_vals`.

Reply:
imo this is clear from the context
anyways, cheap to add, so I did